### PR TITLE
Astro 8/29 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

**Review by ryanahamilton — APPROVED** [review]
I had actually opened PR #2746 proposing these edits (except I missed a couple), but no one reviewed/commented on the PR. Will close it.

File: astro/release-notes.md [hunk 1]
```diff
@@ -27,6 +27,29 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## August 29, 2023
+
+### Changes to Workspace user roles
```
Comment (chomipi88) [inline astro/release-notes.md:32]: Let's also instruct folks to upgrade to the latest CLI release to leverage the new Author role.

**Review by chomipi88 — APPROVED** [review]

File: astro/release-notes.md [hunk 2]
```diff
@@ -27,6 +27,29 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+To increase granularity and better serve each user persona on Astro, Workspace roles have been updated with new names and permissions:
```
Comment (jwitz) [inline astro/release-notes.md:34]:
```suggestion
:::caution Upgrade the Astro CLI

To take advantage of these new user roles programmatically. You must [upgrade the Astro CLI](https://docs.astronomer.io/astro/cli/install-cli#upgrade-the-cli) to 1.19 or later.

:::

To increase granularity and better serve each user persona on Astro, Workspace roles have been updated with new names and permissions:
```

**Comment by chomipi88** [comment]
@jwitz if such scripts exist using the existing Workspace role names, then yes there would be impact.

**Review by lzdanski — APPROVED** [review]
Checked changes in preview - grammar, content LGTM.

**Comment by jwitz** [comment]
Rolling back analytics page changes as those are going into the Thursday release.

**Pull request merged**